snap/bin/launch-multipassd

```diff
@@ -3,4 +3,12 @@
 export http_proxy="$(snapctl get proxy.http)"
 export https_proxy="$(snapctl get proxy.https)"
 
+vm_backend="$(snapctl get vm-backend)"
+
+if [ "$vm_backend" = libvirt ]; then
+    export MULTIPASS_VM_BACKEND=LIBVIRT
+elif [ "$vm_backend" = qemu ]; then
+    export MULTIPASS_VM_BACKEND=QEMU
+fi
```

Comment:
Shouldn't this fall back to qemu _unless_ it's "libvirt"?
What's the value on install?

Reply:
On install, $vm_backend is not set and thus MULTIPASS_VM_BACKEND is not set, so in src/platform/platform_linux.cpp, this defaults to use qeum as the backend.

This is only used so we can use `snap set` to change back and forth after install.

Reply:
Well, yeah, I could do that and either use `snap set multipass vm-backend={LIBVIRT|QEMU}`or use the lower case `libvirt`/`qemu` and that would be the env var.

Reply:
I think I'm going to require `LIBVIRT`/`QEMU` on `snap set` and get rid of the lower case choices.  Does that work for you?

Reply:
Ok, uses the direct `snapctl get`\`export` now.